srcpkgs/poedit/template

```diff
@@ -1,24 +1,23 @@
 # Template file for 'poedit'
 pkgname=poedit
-version=2.4.1
+version=2.4.2
 revision=1
 build_style=gnu-configure
 configure_args="--with-wx-config=wx-config-gtk3"
 hostmakedepends="pkg-config"
 makedepends="boost-devel cld2-devel db-devel enchant2-devel gtk+3-devel
- gtkspell3-devel Lucene++-devel wxWidgets-gtk3-devel"
+ gtkspell3-devel libharfbuzz libxml2 Lucene++-devel webkit2gtk
```

Comment:
I don't understand why these were added; the final package doesn't seem to be linked against them, and they aren't the devel version.

Reply:
Re-looking at it, and retesting, may have needed them during build